tutorials/CMakeLists.txt

```diff
@@ -56,7 +56,8 @@ if(NOT ROOT_minuit2_FOUND)
                    fit/minuit2FitBench.C fit/minuit2GausFit.C
                    fit/minuit2GausFit.C fit/NumericalMinimization.C
                    fit/combinedFit.C fit/TestBinomial.C
-                   fit/fitNormSum.C fit/vectorizedFit.C)
+                   fit/fitNormSum.C fit/vectorizedFit.C
+                   tutorials/roostats/rs_bernsteinCorrection.C)
```

Comment:
Hi Philippe, could you please add a justification as to why this test is being added to the veto list? Is it because of multithreading?

Reply:
Because it requires libMinuit2.so to be built. (it is added only "if(NOT ROOT_minuit2_FOUND)"